articles/architecture-scenarios/implementation/b2c/b2c-authorization.md

```diff
@@ -12,26 +12,84 @@ useCase:
 ---
 # Authorization
 
-It’s important to start by distinguishing between Authentication, Authorization, and Access Control. Your Auth0 tenant (the Authorization Server) is typically responsible for Authentication and some or all of Authorization. Access Control however must be the responsibility of the Application or API itself, because access control is almost always contextual: 
+It’s important to start by distinguishing between Authentication, Authorization, and Access Control. Your Auth0 tenant (your Authorization Server) is typically responsible for Authentication and some or all of Authorization. Access Control however must be the responsibility of the Application or API itself, because access control is almost always contextual: 
 
-* **Authentication**: the process of determining if the user is who they say they are.
-* **Authorization**: the proceess of determining what the user is allowed to do in the system.
-* **Access Control**: the process of limiting a user to only the actions permitted, based on a combination of who the user is, what they are allowed to do in the system, and their consent.
+* **Authentication**: the process of determining if a user is who they say they are.
+* **Authorization**: the proceess of determining what is allowed, based on who a user is and the consent they provide.
```

Comment:
```suggestion
* **Authorization**: the proceess of determining what is allowed, based on who a user is, what permissions they have been given, and/or the consent they provide.
```
Not 100% sure if I like the and/or, but I think it applies

Reply:
Agree with the `and/or` - especially in the case of first-party applications